scripts/generate.sh

```diff
@@ -22,27 +23,33 @@ function docker-run() {
     "$(docker build -f "$(basename "$1")" -q .)"
   popd
 }
+if [ "$LANG" = "" ]; then
+  OUT_DIR=examples/go/go-client/helper/rest
+  generate twilio-go
 
-OUT_DIR=examples/go/go-client/helper/rest
-generate twilio-go
+  OUT_DIR=examples/go/go-client/terraform/resources
+  generate terraform-provider-twilio
 
-OUT_DIR=examples/go/go-client/terraform/resources
-generate terraform-provider-twilio
+  # Replace a couple imports in the generated Terraform resource to use local code.
+  for path in api/v2010 flex/v1; do
+    sed -i.bak "s/github.com\/twilio\/twilio-go/go-client\/helper/g" "$OUT_DIR/$path/api_default.go"
+    sed -i.bak "s/github.com\/twilio\/terraform-provider-twilio\/client/go-client\/terraform\/client/g" "$OUT_DIR/$path/api_default.go"
+  done
 
-# Replace a couple imports in the generated Terraform resource to use local code.
-for path in api/v2010 flex/v1; do
-  sed -i.bak "s/github.com\/twilio\/twilio-go/go-client\/helper/g" "$OUT_DIR/$path/api_default.go"
-  sed -i.bak "s/github.com\/twilio\/terraform-provider-twilio\/client/go-client\/terraform\/client/g" "$OUT_DIR/$path/api_default.go"
-done
+  docker-run examples/go/Dockerfile-goimports
 
-docker-run examples/go/Dockerfile-goimports
+  OUT_DIR=examples/csharp/src/Twilio/Rest
+  generate twilio-csharp
 
-OUT_DIR=examples/csharp/src/Twilio/Rest
-generate twilio-csharp
+  OUT_DIR=examples/java/src/main/java/com/twilio/rest
+  generate twilio-java
 
-OUT_DIR=examples/java/src/main/java/com/twilio/rest
-generate twilio-java
+  OUT_DIR=examples/node/lib/rest
+  generate twilio-node --global-property skipFormModel=false
+  docker-run examples/node/Dockerfile-prettier
+fi
 
-OUT_DIR=examples/node/lib/rest
-generate twilio-node --global-property skipFormModel=false
-docker-run examples/node/Dockerfile-prettier
+if [ "$LANG" = "php" ]; then
```

Comment:
Can this be made to check that if `php` exists in `$LANGAUGES` then it should generate? This allows for supporting of testing multiple languages at the same time.